cluster/images/hyperkube/master-multi.json

```diff
@@ -21,12 +29,19 @@
       "command": [
               "/hyperkube",
               "apiserver",
+              "--admission-control=NamespaceLifecycle,SecurityContextDeny,LimitRanger,ServiceAccount,ResourceQuota",
```

Comment:
When you do that, would you mind deprecating `NamespaceAutoProvision` in flavor of `NamespaceLifecycle` as well?